src/middlewared/middlewared/plugins/system.py

```diff
@@ -1470,6 +1471,7 @@ async def do_update(self, data):
 
         return await self.config()
 
+    @rest_api_metadata(explicit_methods=['GET'])
```

Comment:
That sounds a little bit confusing to me then...
We are saying explicit method is GET, but it also works for POST

From my point of view that would be the list of methods i would want this to work

additional methods, or extra methods might be a better name... or have it be a list of all methods it will accept

Reply:
You are right, we should reword it because it is confusing otherwise. I'll make the required changes.